app/components/NavigationTab/NavigationLink.js

```diff
@@ -12,9 +12,16 @@ type Props = {
 };
 
 const NavigationLink = (props: Props) => {
+  // Custom isActive function as NavLink does not compare query params by default
+  const isActive = (match, location) => {
+    const regex = /\/+$/i; // Regex to remove trailing / for comparison
+    const comparePath = location.pathname + location.search;
+    return props.to.replace(regex, '') === comparePath.replace(regex, '');
```

Comment:
If there were multuple query params, they could be in the wrong order, and this would return a false negative. This is very much an edge case though, and I'm pretty sure it won't happen in lego-webapp, so might be better just to keep it simple and leave it like this.